app/views/csv_files/_csv_file.html.erb

```diff
@@ -0,0 +1,18 @@
+<tr>
+	<td><%= csv_file.id %></td>
+	<td><%= csv_file.name %></td>
+	<td><%= csv_file.humanize_type %></td>
+	<td><%= csv_file.humanize_date %></td>
+
+  <td>
+  	<%= link_to 'Show', csv_file %>
+  </td>
+  <td>
+  	<%= link_to 'Destroy', csv_file, method: :delete, data: { confirm: 'Are you sure?' } %>
```

Comment:
I think I recall that, only I am not sure what we settled on. I think it was in the context of the destroy would not be visible to non "admin" users, though there are no longer roles in the immediate future. I really am not sure what we settled on.